plugins/notifications-common/src/types.ts

```diff
@@ -1,45 +1,66 @@
-export type CreateNotificationRequest = {
-  origin: string;
-  title: string;
-  message?: string;
-  actions?: { title: string; url: string }[];
-  topic?: string;
-  targetUsers?: string[];
-  targetGroups?: string[];
-};
-
 export type NotificationAction = {
   id: string; // UUID
   title: string;
   url: string;
 };
 
-// Keep in sync with FE: plugins/notifications-frontend/src/api/notificationsApi.ts
 export type Notification = {
   id: string; // UUID
   created: Date;
-
-  isSystem: boolean;
   readByUser: boolean;
+  isSystem: boolean;
 
-  origin: string;
+  origin: string; // mandatory
```

Comment:
If we have the '? ' symbol to mark optional then we dont need thecomment, no?

Reply:
good catch, not anymore